### PR TITLE
usibility design 

### PR DESCRIPTION
**Pull request by santeleifer**

### CODE REVIEW

File: app/frontend/components/customerForm/customerForm.tsx [hunk 1]
```diff
@@ -40,35 +41,40 @@ export const CustomerForm = ({ customer }: CustomerFormProps) => {
...
-            {...register('title', {
-              required: 'Name required',
-              minLength: { value: 3, message: 'At least 3 characters' },
-            })}
-          />
-        </label>
-        <div>
-          <ErrorMessage errors={errors} name="title" as={<span className="text-red-700" />} />
-        </div>
+      <article className="timebook">
```
Comment (PascalHelbig) [inline app/frontend/components/customerForm/customerForm.tsx:44]: Do we really need the CSS class `timebook`?

File: app/frontend/components/teamForm/teamForm.tsx [hunk 2]
```diff
@@ -4,6 +4,7 @@ import { TeamFragment, TeamInput, useTeamCreateMutation, useTeamUpdateMutation }
 import { useForm } from 'react-hook-form'
 import { ErrorMessage } from '@hookform/error-message'
 import { useRouter } from 'next/router'
+import { route } from 'next/dist/server/router'
```
Comment (PascalHelbig) [inline app/frontend/components/teamForm/teamForm.tsx:7]: The import looks wrong.

File: app/frontend/components/topNavigation/profileMenu.tsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
+import { Menu } from '@headlessui/react'
+import { signIn, signOut, useSession } from 'next-auth/react'
+import { useRouter } from 'next/router'
+
+import { TopNavigationLink } from './topNavigationLink'
+import Image from 'next/image'
+import { ReactNode } from 'react'
+import Link from 'next/link'
+
+function MyLink(props: { [x: string]: unknown; href: string; children: ReactNode }) {
```
Comment (PascalHelbig) [inline app/frontend/components/topNavigation/profileMenu.tsx:10]: Please change this to an arrow function 😉

Comment (PascalHelbig) [reply app/frontend/components/topNavigation/profileMenu.tsx:10]: And also one component per file...

File: app/frontend/components/topNavigation/profileMenu.tsx [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+      <Menu.Button className="">
+        {session.data?.user.image && (
+          <Image
+            className="rounded-full"
+            width={30}
+            height={30}
+            src={session.data?.user.image}
+            alt={session.data?.user.name ?? 'Profile picture'}
+          />
+        )}{' '}
```
Comment (PascalHelbig) [inline app/frontend/components/topNavigation/profileMenu.tsx:41]:
```suggestion
        )}
```

File: app/frontend/components/topNavigation/profileMenu.tsx [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+      </Menu.Button>
+
+      <Menu.Items className=" absolute right-1 mx-3 flex w-64 flex-col rounded-md bg-white p-2 px-1 shadow ">
+        {teamSlug && (
+          <Menu.Item>
+            <MyLink href={`/${teamSlug}/me`}>Profile</MyLink>
+          </Menu.Item>
+        )}
+        <Menu.Item>
+          <MyLink href={`/time`}>My Timetable</MyLink>
```
Comment (PascalHelbig) [inline app/frontend/components/topNavigation/profileMenu.tsx:51]: This is a bit simpler: 

```suggestion
          <MyLink href="/time">My Timetable</MyLink>
```

File: app/frontend/components/topNavigation/profileMenu.tsx [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+        </Menu.Item>
+        {teamSlug && (
+          <Menu.Item>
+            <MyLink href={`/team`}>Switch Team</MyLink>
```
Comment (PascalHelbig) [inline app/frontend/components/topNavigation/profileMenu.tsx:55]:
```suggestion
            <MyLink href="/team">Switch Team</MyLink>
```

File: app/frontend/components/topNavigation/profileMenu.tsx [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+      </Menu.Items>
+    </Menu>
+  )
+}
+
+/*{({ active }) => (
+  <MyLink className={`${active && 'bg-blue-500'}`} href="/team">
+  Create new team
+</MyLink>
+)}*/
```
Comment (PascalHelbig) [inline app/frontend/components/topNavigation/profileMenu.tsx:74]: This can be deleted

File: app/frontend/styles/globals.css [hunk 8]
```diff
@@ -3,12 +3,19 @@
 @tailwind components;
 @tailwind utilities;
 
+
+html footer {@apply bg-gray-500}
+
+html body{ @apply bg-gray-200}
+
+div.timebook {@apply my-10 rounded-xl border-2 border }
+
```
Comment (PascalHelbig) [inline app/frontend/styles/globals.css:12]: Our plan was to keep this file as small as possible.
Can we move this code to where it is used?

File: app/package.json [hunk 9]
```diff
@@ -34,6 +34,7 @@
     "react-hook-form": "^7.25.1",
     "react-icons": "^4.3.1",
     "react-input-mask": "^3.0.0-alpha.2",
+    "react-share": "^4.4.0",
```
Comment (PascalHelbig) [inline app/package.json:37]: What is that?

File: app/pages/[teamSlug]/team/index.page.tsx [hunk 10]
```diff
@@ -26,43 +26,45 @@ const Team = (): JSX.Element => {
...
-              ))}
-            </TableBody>
-          </Table>
-          <Button variant="primary">Invite Member</Button>
-        </article>
-        <article className="timebook">
-          <h2>Customers</h2>
-          <CustomerTable />
-        </article>
+        <div className="timebook">
```
Comment (PascalHelbig) [inline app/pages/[teamSlug]/team/index.page.tsx:29]: Do we need this `div`?

File: app/pages/team.page.tsx [hunk 11]
```diff
@@ -1,13 +1,23 @@
...
+import { TeamChoiceForm } from '../frontend/components/teamchoice/teamchoice'
 
 const Team = (): JSX.Element => {
   return (
     <ProtectedPage>
-      <article className="timebook">
-        <h2>Create new team</h2>
-        <TeamForm />
-      </article>
+      <div className="timebook">
```
Comment (PascalHelbig) [inline app/pages/team.page.tsx:8]: Do we need this div?

File: app/frontend/components/teamchoice/teamchoice.tsx [hunk 12]
```diff
@@ -0,0 +1,36 @@
+import { useRouter } from 'next/router'
```
Comment (PascalHelbig) [inline app/frontend/components/teamchoice/teamchoice.tsx:1]: The folder and file names must be camelCase: `TeamChoice`

File: app/tailwind.config.js [hunk 13]
```diff
@@ -1,8 +1,14 @@
 /* eslint-disable unicorn/prefer-module */
 
+
 module.exports = {
+ 
   content: ['./pages/**/*.{js,ts,jsx,tsx}', './frontend/**/*.{js,ts,jsx,tsx}'],
   theme: {
+    fontFamily: {
+    'monospaced':'Montserrat' },
```
Comment (PascalHelbig) [inline app/tailwind.config.js:9]: The key should be: `mono` not `monospaced`

Also, we should keep the default values as backup: https://tailwindcss.com/docs/font-family#customizing-the-default-font

And try to install the font: https://github.com/fontsource/fontsource/tree/main/fonts/google/montserrat#readme

**Review by moncapitaine — APPROVED** [review]

**Pull request merged**